smart-contracts/test/helpers/denoms.js

```diff
@@ -54,6 +54,12 @@ const DENOM_4 = generateDenom({
   isERC20: true,
 });
 
+const IBC_DENOM = generateDenom({
```

Comment:
This appears to be ETH, why does it say IBC denom?

Reply:
you can check generateDenom method, if isERC20 as false, it will generate a ibc denom